homeassistant/components/tractive/sensor.py

```diff
@@ -82,7 +89,9 @@ def __init__(
     @callback
     def handle_status_update(self, event: dict[str, Any]) -> None:
         """Handle status update."""
-        self._attr_native_value = event[self.entity_description.key]
+        self._attr_native_value = self.entity_description.value_fn(
```

Comment:
`value_fn` is still missing from already existing sensors

Reply:
`value_fn` has a default value `lambda state: state` and this default value is added to each sensor entity

![obraz](https://github.com/home-assistant/core/assets/478555/e27ce187-15a0-4ad6-af1c-7d59921e8b7d)

Reply:
Right. Didn't notice it has a default. Thanks

Reply:
No worries